guideforms.py

```diff
@@ -53,14 +51,12 @@
 
     'summary': forms.CharField(
         required=True,
-        widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 500}),
+        widget=forms.Textarea(
+            attrs={'cols': 50, 'maxlength': 500, 'placeholder': models.SUMMARY_PLACEHOLDER_TXT}),
```

Comment:
This needs to be models.FeatureForm.SUMMARY_PLACEHOLDER_TXT.
Sorry to give the wrong example before.